tests/Controller/CRUDControllerTest.php

```diff
@@ -1171,6 +1173,17 @@ public function testDeleteActionAjaxSuccess1(): void
             ->with($this->equalTo('delete'))
             ->willReturn(true);
 
+        $formBuilder = $this->createMock(FormBuilderInterface::class);
+
+        $this->admin->expects($this->once())
+            ->method('getFormBuilder')
+            ->willReturn($formBuilder);
+
+        $formBuilder->expects($this->once())
+            ->method('getOption')
+            ->with('csrf_protection')
+            ->willReturn(true);
+
```

Comment:
Have you considered doing this in `setUp()`? Things are getting a bit repetitive…

Reply:
Moved to `setUp` what was possible.